client/src/views/ChallengeView.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div id="header-container" class="text-white">
+        <h1 class="text-md-h1 mt-10 mb-12 font-weight-medium" id="title">PICK A CHALLENGE</h1>
+
+        <div class="ma-10">
+            <h2 style="margin-bottom: 1vh;" class="text-h6 font-weight-bold">Choose your challenge to complete over the 12 days of the
+                challenge</h2>
+            <h2 class="text-h6">(Don't worry if you can't hit your daily target, just do your best!)</h2>
+        </div>
+    </div>
+    <v-sheet class="d-flex flex-wrap justify-center">
+        <v-card class="mx-auto challenge-card pa-5 card-outer" v-bind:style="{ backgroundImage: 'linear-gradient(rgba(0, 0, 0, 0.6),rgba(0, 0, 0, 0.6)), url(' + challenge.img + ')' }" theme="dark" v-for="challenge in challenges">
+            <v-card-title class="text-h4 font-weight-medium py-2" id="card-head">
+                {{ challenge.title }}
+            </v-card-title>
+            <v-card-text class="text-subtitle-1 font-weight-medium py-2">
+                {{challenge.about}}
+            </v-card-text>
+            <v-card-actions id="bottom">
+                <v-list-item class="w-100 pa-6">
+                    <template v-slot:append>
+                        <div>
+                            <v-icon class="me-1" icon="mdi-map-marker-distance"></v-icon>
+                            <span class="text-h7 me-2">{{challenge.totalkm + "km"}}</span>
+                            <span class="text-h7 me-1">·</span>
+                            <v-icon class="me-1" icon="mdi-calendar"></v-icon>
+                            <span class="text-h7">{{challenge.km + "km per day"}}</span>
+                        </div>
+                    </template>
+                </v-list-item>
+            </v-card-actions>
+        </v-card>
+
+    </v-sheet>
+</template>
+
+<script setup lang="ts">
+const challenges = [
+    { title: "WOORABINDA CHALLENGE", img: "/images/Woorabinda.jpg", about: "Virtually travel the distance from one end of Woorabinda to the other.", km: "2", totalkm: "24" },
+    { title: "WURRUMIYANGA CHALLENGE", img: "/images/Wurrumiyanga.jpg", about: "Virtually travel the distance around Wurrumiyanga.", km: "5", totalkm: "60" },
+    { title: "GALIWIN'KU CHALLENGE", img: "/images/Galiwin_ku.jpg", about: "Virtually travel the the distance around Galiwin’ku.", km: "7", totalkm: "84" },
+    { title: "PALM ISLAND CHALLENGE", img: "/images/Palm_Island.jpg", about: "Virtually travel the length of Palm Island.", km: "10", totalkm: "120" }
+]
+
+</script>
+  
+<style>
+@media only screen and (max-width: 800px) {
+    .challenge-card {
+        width: 80vw;
+        margin: 2.5vw;
+    }
+}
+
+@media only screen and (min-width: 800px) {
+    .challenge-card {
+        width: 40vw;
+        margin: 2vh;
+    }
+}
+
+#title {
+    font-family: 'Hackney' !important;
+}
+
+.challenge-image {
+    width: 1000
+}
+
+.challenge-card {
+    box-shadow: 0 2px 7px rgb(0 0 0 / 0.1);
+    height: 35vh;
```

Comment:
Try to make the cards more rectangular (ie shorter) to align more closely with the figma